src/parsers/texttracks/ttml/html/create_element.ts

```diff
@@ -462,7 +462,7 @@ function generateTextContent(
  * @param {Array.<Object>} regions
  * @param {Array.<Object>} styles
  * @param {Object} paragraphStyle
- * @param {Boolean} shouldTrimWhiteSpaceOnParagraph
+ * @param {Boolean} shouldTrimWhiteSpace
```

Comment:
Better, but in reality the last argument is an Object (which contains `shouldTrimWhiteSpace`)